### ISSUE TITLE
Personalization toolbar and bottom bar colors

### ISSUE THREAD

**Issue by mortinus** [dirkwhoffmann/virtualc64#402]
Is it possible to change the color nuances of the title / toolbar and bottom bar with the color of the old commodore64 colors and the commodore 64C newest colors?
... Two new themes other this metal / gray etc ... for those who like the antique style ...

**Comment by UgoCaneFifone** [comment]
Excellent idea, this is a great idea!
While playing, seeing that Commodore light brown in place of metal is nobler

**Comment by Alessandro1970** [comment]
I made this toolbar from a picture of a C64.
The Status Bar could be, in solid color, of the same color. It would serve the right contrast for the icons.

![untitled 3](https://user-images.githubusercontent.com/34916641/44343188-90b62400-a48d-11e8-834a-d6679ed30fd4.png)

**Comment by dirkwhoffmann** [comment]
I don't think it's a good idea to customize the macOS specific UI items. Apple itself did this a couple of years ago when they used a leather texture for address book and some other fancy looking textures for game center etc. They skipped this idea in favor of a consistent looking desktop and in my personal opinion, they were right about this. I don't think personally that an emulator needs an old fashioned look and that it's better to let it behave like a good macOS app citizen (meaning to accept OS related UI standards). The same applies to the title bar (which shows 'Untitled' most of the time). This is due to the fact that the emulator is a document based OS X app with Snapshots as it's document type. 

Icons are another story. It might indeed give the app a nicer look and feel if the artwork was a little bit more retro. However, as I posted before, it is quite difficult to find a set of consistently looking icons. Graphic design is an often underestimated profession, because great looking artwork is usually unobtrusive. A good example is macOS itself (i.e. prior to Yosemite in my opinion). The first time I had noticed how well Apple designers worked is when I compared it with Linux. I simply liked the Linux artwork a little less, but when I looked at a specific icon in detail, I was not able to say why. This is what I mean when I'm saying that graphic design is often underestimated.
